### ISSUE TITLE
Unable to execute script of NVDA Dev & Test Toolbox

### ISSUE THREAD

**Issue by CyrilleB79** [javidominguez/commandHelper#11]
This add-on is able to list the script of NVDA Dev & Test Toolbox but cannot execute them.

The following error is in the log:
```
ERROR - scriptHandler.executeScript (23:10:00.181) - MainThread (6892):
error executing script: <bound method GlobalPlugin.script_executeCommand of GlobalPlugin ('globalPlugins.commandHelper')> with gesture 'entrée'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 295, in executeScript
  File "C:\Users\Cyrille\AppData\Roaming\nvda\addons\commandHelper\globalPlugins\commandHelper\__init__.py", line 398, in script_executeCommand
    i = [m.__module__ for m in globalPluginHandler .runningPlugins].index(commandInfo.moduleName)
ValueError: 'globalPlugins.ndtt.stackTracing' is not in list
```

I am using:
* Command Helper v dev20230730
* NVDA 2023.2rc1
* Windows 10 2004 (AMD64) build 19041.388

Could you please have a look?
